coding_sheets/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class ProcessedSheet(models.Model):
+    country = models.CharField(max_length=255)
+    sheet_name = models.CharField(max_length=255)
+    sheet_tab = models.CharField(max_length=255)
+    sheet_row = models.IntegerField()
```

Comment:
If we're storing even rows, shouldn't this model be called `ProcessedSheetRow` then?

Reply:
sure. Sounds better.